### PR TITLE
[syncGeneratedFilesWrapper] Fix syncGeneratedFilesWrapper

### PR DESCRIPTION
**Pull request by DmitriiBobreshev**
**Description**: 
- Fixed cases when the generated files already were in changes

**Documentation changes required:**N

**Added unit tests:** N

**Checklist**:
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: make-util.js [hunk 1]
```diff
@@ -1895,65 +1895,49 @@ exports.generateTasks = generateTasks;
...
             let dest = path.join(__dirname, 'Tasks', baseTaskName, fileName);
             if (config) {
                 dest = path.join(__dirname, 'Tasks', baseTaskName, '_buildConfigs', config, fileName);
             }
 
-            fs.copyFileSync(filePath, dest);
-            console.log(`from ${filePath} to ${dest}`);
+            // Prefer execSync instead of run, because gir diff throw error if there is a diff between files
+            try {
+                ncp.execSync(`git diff --no-index ${filePath} ${dest} --numstat`);
```
Comment (merlynomsft) [inline make-util.js:1930]: >t diff [](http://example.com/codeflow?start=32&length=6)

minor: I'm thinking the diff isn't necessary and we can just copy; if the files are identical, git will ignore the change

**Review by merlynomsft — DISMISSED** [review]
:shipit:

File: make-util.js [hunk 2]
```diff
@@ -1895,65 +1895,49 @@ exports.generateTasks = generateTasks;
...
-            `${changedPaths}\n` +
-            "----------------------------------------------\n" +
-            "The following files were copied to source task:");
-
-        changedPaths.forEach((filePath) => {
-            const fileName = path.basename(filePath);
-            if (allowedFilesToCopy.indexOf(fileName) === -1) return;
-            const [baseTaskName, config] = taskName.split("_");
+        runtimeChangedFiles.forEach((fileName) => {
+            const filePath = path.join(genTaskPath, fileName);
```
Comment (merlynomsft) [inline make-util.js:1920]: >filePath = path.join(genTaskPath, fileN [](http://example.com/codeflow?start=18&length=39)

I believe this misses files in subdirectories e.g. Test\package-lock.json

Comment (DmitriiBobreshev) [reply make-util.js]: Fixed, thanks

**Review by merlynomsft — APPROVED** [review]

File: make.js [hunk 3]
```diff
@@ -221,6 +221,7 @@ CLI.serverBuild = function(/** @type {{ task: string }} */ argv) {
         allTasksNode20.forEach(taskName => buildTaskWrapped(taskName, allTasksNode20.length, 20));
 
     } 
+
```
Comment (merlynomsft) [inline make.js:224]: whitespace only

**Comment by merlynomsft** [comment]
FYI - I cherrypicked this change here: #19008

File: make.js (hunk 3 above)
Comment (DmitriiBobreshev) [reply make.js]: removed, thanks!

**Pull request merged**